### PR TITLE
[ja] Fix kubernetes-api.md

### PR DESCRIPTION
**Pull request by eltociear**
markdown format

### CODE REVIEW

**Review by avinashupadhya99 — APPROVED** [review]

File: content/ja/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Kubernetes APIサーバーは、`/openapi/v2`エンドポイントを介してOp
 </table>
 
 
-Kubernetesは、他の手段として主にクラスター間の連携用途向けのAPIに、Protocol buffersをベースにしたシリアライズフォーマットを実装しています。このフォーマットに関しては、[Kubernetes Protobuf serialization](https://github.com/kubernetes/community/blob/master/contributors/des ign-proposals/api-machinery/protobuf.md)デザイン提案を参照してください。また、各スキーマのInterface Definition Language（IDL）ファイルは、APIオブジェクトを定義しているGoパッケージ内に配置されています。
+Kubernetesは、他の手段として主にクラスター間の連携用途向けのAPIに、Protocol buffersをベースにしたシリアライズフォーマットを実装しています。このフォーマットに関しては、[Kubernetes Protobuf serialization](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/protobuf.md)デザイン提案を参照してください。また、各スキーマのInterface Definition Language（IDL）ファイルは、APIオブジェクトを定義しているGoパッケージ内に配置されています。
```
Comment (riita10069) [inline content/ja/docs/concepts/overview/kubernetes-api.md:68]: I accessed this link and it seems to be archived.

https://github.com/kubernetes/design-proposals-archive/blob/main/api-machinery/protobuf.md
I think this one is better.

Comment (riita10069) [inline content/ja/docs/concepts/overview/kubernetes-api.md:68]:
```suggestion
Kubernetesは、他の手段として主にクラスター間の連携用途向けのAPIに、Protocol buffersをベースにしたシリアライズフォーマットを実装しています。このフォーマットに関しては、[Kubernetes Protobuf serialization](https://github.com/kubernetes/design-proposals-archive/blob/main/api-machinery/protobuf.md)デザイン提案を参照してください。また、各スキーマのInterface Definition Language（IDL）ファイルは、APIオブジェクトを定義しているGoパッケージ内に配置されています。
```

**Comment by eltociear** [comment]
@riita10069 THX!

**Comment by nasa9084** [comment]
Thank you for your contribution~
/approve

**Pull request merged**